SPECS/emu-manager.spec

```diff
@@ -42,6 +42,14 @@ cd build
 %{_libdir}/xen/bin/emu-manager
 
 %changelog
+* Fri Oct 21 2022 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.2.0-1
+- New version 1.2.0
+- Streams support pipes
+
+* Fri Sep 09 2022 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.1.4-1
+- New version 1.1.4
+- Correctly report error codes when opening an emu stream using a non-sock file
+
```

Comment:
We don't add changelog entries for versions that are not built in the build system. You can simply add the last line to the new changelog entry.